ReservePubKey/handler.ts

```diff
@@ -98,7 +92,13 @@ export const reservePubKeys = (
       flow(
         R.filter(isDefined),
         R.map(reserveSingleKey(lollipopPubkeysModel, inputPubkeys.pub_key)),
-        A.sequenceS(TE.ApplicativePar)
+        A.sequenceS(TE.ApplicativePar),
+        eventLog.taskEither.errorLeft(error => [
+          `Error reserving keys: ${error.detail}`,
+          {
+            name: FN_LOG_NAME
```

Comment:
I can add the keys to the event, sure. But how do you want to verify if both of them failed? If we want to check them programmatically, we need to refactor the code, since `A.sequenceS(TE.ApplicativePar)` is returning just one error.

Reply:
Keys value added in [a14d8d7](https://github.com/pagopa/io-functions-lollipop/pull/36/commits/a14d8d78391a5f98b6d525db50d3b2db0e393a74)

Reply:
The whole operation fails. We need only to add information about the failure. The check can be performed manually. But the TTL of the key is 15 minutes for this scenario, so there is an high probability that a manual check doesn't find any key on the database.